<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputAddEditRemove</class>
 <widget class="QWidget" name="inputAddEditRemove">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>28</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">inputAddEditRemove</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="btnAdd">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../ressources.qrc">
       <normaloff>:/rec/data/icons/iconAdd.svg</normaloff>:/rec/data/icons/iconAdd.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnEdit">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../ressources.qrc">
       <normaloff>:/rec/data/icons/iconBroom.svg</normaloff>:/rec/data/icons/iconBroom.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnRemove">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../ressources.qrc">
       <normaloff>:/rec/data/icons/iconDelete.svg</normaloff>:/rec/data/icons/iconDelete.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>56</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
